compiler/compile/OMRSymbolReferenceTable.cpp

```diff
@@ -1899,3 +1900,57 @@ TR::SymbolReference * & OMR::SymbolReferenceTable::element(CommonNonhelperSymbol
       return nullSymRef;
       }
    }
+
+void OMR::SymbolReferenceTable::makeSharedAliases(TR::SymbolReference *sr1, TR::SymbolReference *sr2)
+   {
+   int32_t symRefNum1 = sr1->getReferenceNumber();
+   int32_t symRefNum2 = sr2->getReferenceNumber();
+   TR_BitVector *aliases1 = NULL;
+   TR_BitVector *aliases2 = NULL;
+
+   if (_sharedAliasMap == NULL)
+      {
+      _sharedAliasMap = new (comp()->trHeapMemory()) AliasMap(std::less<int32_t>(), getTypedAllocator<std::pair<int32_t, TR_BitVector *> >(comp()->allocator()));
```

Comment:
If I'm reading this correctly and this is per symbol reference table, why not make this a real member instead of a pointer allocated on demand?

Also, you shouldn't need "getTypedAllocator" anymore.  If you pass comp()->allocator() directly, it should get converted into the right kind of typed_allocator automatically, thanks to a conversion operator I added to CS2::shared_allocator.